### ISSUE TITLE
Better to mount ssh key, than bake it into the image

### ISSUE THREAD

**Issue by bulldog98** [OkieOth/kubectl.ssh.example#1]
If I understood your Dockerfile correctly you bake the ssh key into the image, proper way to get ssh key into an image is to mount that as a volume

**Comment by OkieOth** [comment]
I agree :D ... tried to use it in a helm chart and run into problems ... next version will tate it from base64 encoded env vars. That's more dynamic at runtime ... hopefully ;)
